modules/nw-router-configuring-dual-stack.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-router-configuring-dual-stack_{context}"]
+= Configuring the {product-title} Ingress Controller for dual-stack support
+
+When you install a {product-title} cluster on a dual-stack enabled bare metal infrastructure, you can configure the Ingress Controller for single-stack or dual-stack support.
+
+.Prerequisites
+
+* You deployed an {product-title} cluster on bare metal.
+* You installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+To configure the Ingress Controller for single-stack or dual-stack support, create or modify a service by setting the `ipFamilies` and `ipFamilyPolicy` fields.
```

Comment:
Creating a service with `ipFamilies` set doesn't technically configure the ingress controller. Maybe instead you could say "in order to have the ingress controller serve traffic over ipv6/ipv4 to a user workload....create or modify..." or something more along those lines?